05 - VerySimpleFixedPVC2_with_electrical_layout.pvc

```diff
@@ -278,7 +278,10 @@
 	<library_nodes>
 		<!-- We define the model for a table with 1 rack -->
 		<node id="TableModel1" sid="TableModel1">
-			<!-- We instantiate the rack model -->
+			<!-- We instantiate the rack model -->			
+			<rotate sid="azimuth">0 0 1 45</rotate>
+			<rotate sid="base slope">0 1 0 10</rotate>
+			<rotate sid="tilt">1 0 0 21.8</rotate>

```

Comment:
`tilt`, `base_slope` and `azimuth` are `<rotation>` elements recognized by collada.

Benefits:
- they are understood by a non-pv 3d software
- PV software can easily read the last number and based on the `sid` indentify that `tilt` is `21.8`, `azimuth` is `45` and `base_slope` is `10`

Notes:
- the order is important
- the axes (first three numbers in the <rotate> element), are constant